antiplag/models.py

```diff
@@ -64,20 +66,23 @@ def process_file(self):
 
     def process_raw_text(self):
         # Replace with actual process raw text method.
-        return 'text'
+        return self.text_raw
 
     @staticmethod
     def detect_language(text_raw):
         return detect(text_raw)
 
 
 class Result(models.Model):
-    document = models.OneToOneField(Document, on_delete=models.CASCADE, related_name='result')
+    document = models.OneToOneField(
+        Document, on_delete=models.CASCADE, related_name="result"
+    )
     matched_docs = models.JSONField()
     percentage = models.FloatField(default=0)
 
     def __str__(self):
         return f"{self.document}"
 
+    @property
```

Comment:
What does this decorator do? Shouldn't the serializer which uses it be updated too?

Reply:
It turns python class method into class attribute. Function now acts as a getter.

Looks like serializer works just fine and doesn't care whether you pass a function or an attribute.

Reply:
Thanks for the explanation.